app/models/opentofu_worker.rb

```diff
@@ -76,7 +76,7 @@ def unit_environment_variables
       "PORT"                  => container_port,
       "OPENTOFU_RUNNER_IMAGE" => container_image,
       "LOG4JS_LEVEL"          => ::Settings.log.level_embedded_terraform,
-      "TF_OFFLINE"            => worker_settings[:opentofu_offline]
+      "TF_OFFLINE"            => worker_settings[:opentofu_offline].to_s
```

Comment:
This one isn't strictly necessary because when we write these out to the override file we are doing a string interpolation anyway

Reply:
I think it's still good to have as:
1. It keeps the code in sync with the containerized code.
2. The Terraform runner always expects it in all lowercase, which `.to_s` already does.

Reply:
Done